vaadin-date-picker-flow-parent/vaadin-date-picker-flow/src/test/java/com/vaadin/flow/component/datepicker/DatePickerTest.java

```diff
@@ -216,4 +217,28 @@ public void setDateFormats_throwsExceptionWhenSecondArgIsNull() {
                 () -> i18n.setDateFormats("MM-yyyy-dd", null));
     }
 
+    @Test(expected = IllegalArgumentException.class)
+    public void datePickerFirstDayOfTheWeek() {
+        DatePicker germanDatePicker = new DatePicker();
+        germanDatePicker.setLabel("German");
+        germanDatePicker.setValue(LocalDate.now());
+
+        DatePickerI18n datePickerI18n = new DatePickerI18n();
+        datePickerI18n.setWeek("Woche");
+        datePickerI18n.setCalendar("Kalender");
+        datePickerI18n.setClear("Löschen");
+        datePickerI18n.setToday("Heute");
+        datePickerI18n.setCancel("Abbrechen");
+        datePickerI18n.setWeekdays(Arrays.asList("Sonntag", "Montag",
```

Comment:
The build is failing because the compiler can't find `Arrays` symbol. Add the corresponding import 
```java
import java.util.Arrays;
```
And it's good.

Reply:
Thanks for noting